documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -28,97 +28,101 @@ We request OpenAPI(Swagger) spec authoring be assigned to engineers who have an
 
 #### RPC Errors
 
-| Id | Rule Name |
-| --- | --- |
-| [R3012](#R3012)	| [APIVersionPattern](#R3012)	|
-| [R3019](#R3019)	| [ARMResourcePropertiesBag](#R3019)	|
-| [R3014](#R3014)	| [BodyPropertiesNamesCamelCase](#R3014) |
-| [R3016](#R3016)	| [DefinitionsPropertiesNamesCamelCase](#R3016)  |
-| [R3006](#R3006)	| [BodyTopLevelProperties](#R3006) |
-| [R3008](#R3008)	| [CollectionObjectPropertiesNaming](#R3008) |
-| [R2044](#R2044)	| [InvalidVerbUsed](#R2044) |
-| [R3023](#R3023)	| [OperationsAPIImplementation](#R3023) |
-| [R3007](#R3007)	| [PutGetPatchResponseSchema](#R3007) |
-| [R3025](#R3025)	| [TrackedResourceGetOperation](#R3025) |
-| [R3026](#R3026)	| [TrackedResourcePatchOperation](#R3026) |
-| [R2059](#R2059)	| [UniqueResourcePaths](#R2059) |
-| [R2016](#R2016)	| [PatchBodyParametersSchema](#R2016) |
-| [R2062](#R2062)	| [XmsResourceInPutResponse](#R2062) |
-| [R3027](#R3027)	| [TrackedResourceListByResourceGroup](#R3027) |
-| [R3028](#R3028)	| [TrackedResourceListBySubscription](#R3028) |
-| [R3011](#R3011)	| [DescriptionMustNotBeNodeName](#R3011) |
-| [R2020](#R2020) | [RequiredPropertiesMissingInResourceModel](#R2020) |
+| Id | Rule Name | Applies to |
+| --- | --- | --- |
+| [R3012](#R3012)	| [APIVersionPattern](#R3012)	| ARM Swagger specs |
+| [R3019](#R3019)	| [ARMResourcePropertiesBag](#R3019)	| ARM and Data plane Swagger specs |
+| [R3014](#R3014)	| [BodyPropertiesNamesCamelCase](#R3014) | ARM and Data plane Swagger specs |
+| [R3016](#R3016)	| [DefinitionsPropertiesNamesCamelCase](#R3016)  | ARM and Data plane Swagger specs |
+| [R3006](#R3006)	| [BodyTopLevelProperties](#R3006) | ARM Swagger specs |
+| [R3008](#R3008)	| [CollectionObjectPropertiesNaming](#R3008) | ARM and Data plane Swagger specs |
+| [R2044](#R2044)	| [InvalidVerbUsed](#R2044) | ARM and Data plane Swagger specs |
+| [R3023](#R3023)	| [OperationsAPIImplementation](#R3023) | ARM Swagger specs |
```

Comment:
I am not sure at this point. It would require service side changes. I will follow up with Gaurav to understand how it helps and how it works to decide if we want to push the same for data plane specs.